lighthouse-cli/run.js

```diff
@@ -27,12 +27,25 @@ const _PROTOCOL_TIMEOUT_EXIT_CODE = 67;
 
 /**
  * exported for testing
- * @param {string} flags
+ * @param {string|Array<string>} flags
  * @return {Array<string>}
  */
 function parseChromeFlags(flags = '') {
-  const parsed = yargsParser(
-      flags.trim(), {configuration: {'camel-case-expansion': false, 'boolean-negation': false}});
+  // flags will be a string if there is only one chrome-flag parameter:
+  // i.e. `lighthouse --chrome-flags="--user-agent='My Agent' --headless"`
+  // flags will be an array if there are multiple chrome-flags parameters
+  // i.e. `lighthouse --chrome-flags="--user-agent='My Agent'" --chrome-flags="--headless"`
+  const trimmedFlags = (Array.isArray(flags) ? flags : [flags])
+      // `child_process.exceFile` wrappes quotes arround command line arguments to escape them
+      // in this case yargs will not be able to parse multiple flags
+      // i.e. `child_process.execFile("lighthouse", ["http://google.com", "--chrome-flags='--headless --no-sandbox'")`
+      // the following regular expression removes those wrapping quotes:
+      .map((flagsGroup) => flagsGroup.replace(/^\s*('|")(.+?)\1?\s*$/, '$2').trim())
```

Comment:
```suggestion
      .map((flagsGroup) => flagsGroup.replace(/^\s*('|")(.+?)\1\s*$/, '$2').trim())
```

wasn't trying to say we should handle that case see below for explanation, sorry for the miscommunication there what you had was fine :)

Reply:
Okay reverted 👍